x-pack/plugins/security_solution/public/management/common/url_routing/utils.ts

```diff
@@ -14,8 +14,8 @@ import { MANAGEMENT_DEFAULT_PAGE_SIZE, MANAGEMENT_PAGE_SIZE_OPTIONS } from '../c
  * @param value
  * @param defaultValue
  */
-export const isDefaultOrMissing = <T>(value: T | undefined, defaultValue: T) => {
-  return value === undefined || value === defaultValue;
+export const isDefaultOrMissing = <T>(value: T | undefined | 0, defaultValue: T) => {
+  return value === undefined || value === defaultValue || value === 0;
```

Comment:
Why did you add `0` here? 😕 

Note: I'm hopping that these new set of utilities here will handle only the "new" type of our standardized URL params - like `page` is `1` based, while pages that still use page index should be using the older routing functions.

Reply:
Yeah, so one of the routing tests was expecting the page to be `0` instead of `1` when the URL params were manually set to the default page (which is 0) or `page=0`. See [here](https://github.com/elastic/kibana/pull/129208/files#diff-6143efd0d047ed76e00616be1b224d3c6b8e302bbf85c01166ef072a73008724R219-R229). I'm assuming, in this case, it should be reset to `1`.

Reply:
we should not be using `page=0` for new pages, as we agreed that going forward they all should be "1-based". We also should not be using `MANAGEMENT_DEFAULT_PAGE` because that holds a page index (0-based) and not a page number (1-based). Actually - we probably don't even need a "DEFULT_PAGE" const (I don't remember why we even added it, but it was when we first created the plugin in Kibana)